src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -2472,6 +2494,15 @@ public void ConfigureForCtrlBackspace(int cursorRelativeToEnd = 0)
         private class ComboBoxWithSelectCounter : ComboBox
         {
             public int EventsCount { get; private set; }
+
+            public void AddItems(int numToAdd)
+            {
+                for (int i = 0; i <= numToAdd; i++)
```

Comment:
Now it adds numToAdd + 1 items
```suggestion
                for (int i = 0; i < numToAdd; i++)
```

Reply:
Though `comboBox.AddItems(expectedKeyPressesCount+1);` would be better. Thus we will be adding the exact number of items given in the argument.